<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualWindow</class>
 <widget class="QMainWindow" name="VisualWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>数据可视化</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="top_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/a/top.jpg);</string>
    </property>
    <property name="text">
     <string>数据可视化      </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>500</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="cloud_groupbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>240</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>云图</string>
    </property>
    <widget class="QLabel" name="wordnum_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>选择课程数（学校数）</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="wordnum_combobox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="schoolcloud_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>44</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开课门数的学校词云</string>
     </property>
    </widget>
    <widget class="QPushButton" name="coursecloud_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>74</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>参加人数的课程词云</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="pie_groupbox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>饼图</string>
    </property>
    <widget class="QLabel" name="school_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>学校名称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="school_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="coursenum_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>课程数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="coursenum_combobox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enrollbar_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>参加人数最多的课程饼图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="evaluatebar_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>评价人数最多的课程饼图</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="top.qrc"/>
 </resources>
 <connections/>
</ui>
